packages/editor/src/lib/editor-wrapper.svelte

```diff
@@ -0,0 +1,39 @@
+<!--
+  Wrap a CodeMirror instance with custom styling.
+  @component
+-->
+<script lang="ts">
+/* eslint-disable tailwindcss/no-custom-classname */
```

Comment:
new line. Otherwise copy-over.

Reply:
Is it complaining about `cm-wrapper` despite being defined below? [This rule](https://github.com/francoismassart/eslint-plugin-tailwindcss/blob/master/docs/rules/no-custom-classname.md) is valid, and we don't want to use a style that is not defined.

Reply:
@ethanlook  Yes, it is complaining about cm-wrapper. Do you have any ideas why this same lint rule isn't triggering in ui?